src/commands/networks/removeNetwork.ts

```diff
@@ -16,6 +16,11 @@ export async function removeNetwork(context: IActionContext, node: NetworkTreeIt
         nodes = await ext.networksTree.showTreeItemPicker(NetworkTreeItem.contextValue, { ...context, canPickMany: true, suppressCreatePick: true });
     }
 
+    if (nodes.some(node => ['bridge', 'host', 'none'].includes(node.networkName))) {
```

Comment:
I wonder if clever use of tree item context values would allow us to filter out the built-in networks from being chosen.  Ideally, they'd be listed but disabled, but I'm not sure the VS Code picker API allows that so just filtering them out completely may be better than warning them after the fact.

We *should* akso be able to remove/disable (preferably the latter) the remove command from the context menus of the built-in nodes.

Reply:
I have added a new commit which filters them out. I could not figure out how to disable the command but still have it visible (in either case). I agree it would be preferable and perhaps somebody more knowledgeable can figure out if that's even possible.

Reply:
I don't think it is in the current framework. In any case, I'm happy with hiding `Remove` entirely from built-in networks.

Reply:
I'll resolve this comment, the current implementation will not show the `Remove` button when right-clicking a built-in network. If you multi-select and do `Remove` on a non-built-in network, it will filter the built-in ones out of the list to prevent errors.